oas_docs/output/kibana.serverless.yaml

```diff
@@ -14146,6 +14146,14 @@ paths:
           required: false
           schema:
             type: string
+        - description: Aggregated gap fill statuses
```

Comment:
I need help understanding the description. Can you explain in a different way what the `gap_fill_statuses` returns?

Reply:
Represents the status of a gap fill process for a rule.
 This is a derived, per-rule aggregation that summarises how well the set of gaps for a rule has been filled. It is calculated from the gap duration sums with priority: unfilled > in_progress > filled.

Reply:
I'm still having trouble understanding what the `gap_fill_statuses` param actually queries and what (if any) values users can specify. The "aggregated gap status" term is still unclear to me, along with the explanation of it in the PR description. A demo of the corresponding UI element (if there is one) would be really helpful here. A sample response for this param would be helpful too.

If the `gap_fill_statuses` param is meant to retrieve a summary of the gap statuses that you specify (e.g., `Filled`, `In progress`, and `Unfilled`), perhaps the following revision would work: 

Array containing statuses that are used to indicate the states of gaps that have been reported for a rule. The values must be: `Filled`, `In progress`, and `Unfilled`